README.md

```diff
@@ -1,24 +1,48 @@
-# README
+# Blog 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is a Blog website
 
-Things you may want to cover:
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on rails
 
-* Configuration
 
-* Database creation
+## Getting Started
 
-* Database initialization
 
-* How to run the test suite
+To get a local copy up and running follow these simple example steps.
 
-* Services (job queues, cache servers, search engines, etc.)
+- Clone the repository to your local machine
+- Open the terminal and navigate to the project directory
 
-* Deployment instructions
 
-* ...
+
+## Authors
+
+👤 **Baraka Danny (Me)**
+
+- GitHub: [@barakadan](https://github.com/barakadanny)
```

Comment:
- [ ] Please use the new [Microverse README template](https://github.com/microverseinc/readme-template) instead of the old one.

Reply:
@Omar-Muhamad 
Thank you for the review,
The changes have been implemented.